MachineLearningPrototype/utils/preprocess_data.py

```diff
@@ -15,3 +16,48 @@ def extract_quotation_content(content):
     if (len(substring) >= 1):
         result=' '.join(substring)
         return result
+
+def query_trss_for_jenkins_output(jenkins_url, test_name, config_file_path="config.json"):
+    #Get TRSS servers to query
+    with open(config_file_path) as config_file:
+        config = json.loads(config_file.read())
+        if "trss_servers" not in config:
+            raise Exception("No TRSS servers in config file")
+            return ""
+        servers = config["trss_servers"]
```

Comment:
For this PR, I think we can put the function's third input to `trss_servers` directly. Then I will open another issue to read config file info in the main function, so sub-functions do not need to read config file separately.

Reply:
Took `trss_servers` as the third input. Thank you!